roles/install_openstack_ca/tasks/main.yml

```diff
@@ -14,12 +14,15 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Get CA bundle data
+- name: Get CA bundle data with retries
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
   ansible.builtin.command:
-    cmd: 'oc get secret combined-ca-bundle -n openstack -o "jsonpath={.data.tls-ca-bundle\.pem}" --ignore-not-found'
```

Comment:
Maybe we should add `ignore_errors: true` to match the behavior of `--ignore-not-found`

Try get the secret a few times to allow it time to create, if it never shows up then allow the task failure to match `--ignore-not-found`

Reply:
thanks @lewisdenny  that's a great idea